packages/hashi/src/SCORM.js

```diff
@@ -622,8 +622,8 @@ export default class SCORM extends BaseShim {
       const raw = Number(isNaN(score.raw) ? min : score.raw);
       return Math.max(Math.min((raw - min) / (max - min), 1), 0);
     }
-    const lessonStatus = this.data.cmi.core.lesson_status;
-    if (statusProgressMap.hasOwnProperty(lessonStatus)) {
+    const lessonStatus = getByKeyPath(this.data, 'cmi.core.lesson_status', SCHEMA, self.userData);
+    if (Object.prototype.hasOwnProperty.call(statusProgressMap, lessonStatus)) {
```

Comment:
Just curious about why this change was made. e.g. how `Object.prototype.hasOwnProperty.call` differs in behavior from `statusProgressMap.hasOwnProperty`. Is this to handle the case where `lessonStatus` is undefined? If so, any reason not to check it for undefined first?

Reply:
This was flagged by our linter which has this rule: https://eslint.org/docs/rules/no-prototype-builtins

I made the same change the last time I merged `pf-mvp` -> `develop`

Reply:
Yeah - I originally made this change on develop, so @indirectlylit's linting fix was caught up in my stash. It seemed simpler to retain this change than to undo it when I reapplied it to pf-mvp.